packages/twenty-server/src/workspace/messaging/repositories/create-companies-and-contacts/create-companies-and-contacts.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@nestjs/common';
+
+import { EntityManager } from 'typeorm';
+
+import { WorkspaceDataSourceService } from 'src/workspace/workspace-datasource/workspace-datasource.service';
+import { Participant } from 'src/workspace/messaging/types/gmail-message';
+import { getDomainNameFromHandle } from 'src/workspace/messaging/services/utils/get-domain-name-from-handle';
+import { CreateCompanyService } from 'src/workspace/messaging/services/create-company/create-company.service';
+import { CreateContactService } from 'src/workspace/messaging/services/create-contact/create-contact.service';
+
+@Injectable()
+export class CreateCompaniesAndContactsService {
+  constructor(
+    private readonly workspaceDataSourceService: WorkspaceDataSourceService,
```

Comment:
Ideally we shouldn't import WorkspaceDataSourceService inside non repository services, raw queries should be moved to repositories instead.